### DISCUSSION TITLE
see maintenance task frequency from list views

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#39] [Shipped - Live on production]
I can see no good option currently to see the frequency of certain maintenance tasks for devices. It would be helpful if the frequency could be shown in one of the lists. At least in the overview of tasks, and if possible also in the device overview (but that might be complicated if there is more than 1 type of maintenance scheduled).

**Comment by YvesR** [comment]
we can not provide this at the grid view, but inside we will shot on selected items.
